clothing/views.py

```diff
@@ -38,6 +41,27 @@ def done(self, form_list, form_dict, **kwargs):
         for form in form_dict.get('1'):
             if 'type' in form.cleaned_data:
                 form.save()
+
+        email_addr = self.get_cleaned_data_for_step("0")['email']
+        person = Person.get_by_email_address_current(email_addr)
+        orders = '\n'.join(o.info() for o in Order.get_current(person=person))
+
+        email = EmailMessage()
+        email.subject = _("Kleiderbestellung %(ophase)s") % {'ophase': str(Ophase.current())}
+        email.body = _("""Hallo %(name)s,
+
+deine Kleiderbestellung für die Ophase wurde aktualisiert.
+Aktuell ist folgendes für dich vermerkt:
+
+%(orders)s
+
+Liebe Grüße
+die Ophasenleitung
+""") % {'name': person.prename, 'orders': orders}
+        email.to = [email_addr]
+        email.reply_to = ["ophase-leitung@fachschaft.informatik.tu-darmstadt.de"]
```

Comment:
this should be set somewhere else. Maybe inside the settings model or inside the ophasebase settings.